### PR TITLE
[12.0][MIG] account product fiscal classification

### PR DESCRIPTION
**Pull request by legalsylvain**
supersed : https://github.com/OCA/account-fiscal-rule/pull/92 
Two commits : 
- https://github.com/OCA/account-fiscal-rule/pull/95/commits/a4b46bd886a89d84d29f9d99e8d66469242bf5a8 work, done by @pierrickBrun
- https://github.com/OCA/account-fiscal-rule/pull/95/commits/9158f0588adf4ea20802fcdabe222ee6ba2b796d to finish to port.

main changes regarding #92

- [FIX] restore the call to the hook, when installing the module. (to populate all the fiscal classification)
- [IMP] link template to account.chart.template So, installing a chart template create fiscal positions. (no more wizard).
- [ADD] possibility to restrict the usage of fiscal classification to a given group. (The typical use case is that some configuration are simple and should be used by all people that can create products, but other configuration are more technical and should be used only by accountants that know what there are doing).

### CODE REVIEW

**Comment by legalsylvain** [comment]
Ready for review. All is green. CC : @PierrickBrun, @quentinDupont

**Comment by legalsylvain** [comment]
@PierrickBrun, could you check the diff ? 

regards.

File: account_product_fiscal_classification/__init__.py [hunk 1]
```diff
@@ -31,19 +31,23 @@ def create_fiscal_classification_from_product_template(cr, registry):
...
         else:
             # associate product template to existing Fiscal Classification
-            template.fiscal_classification_id = classifications_keys.keys()[
-                classifications_keys.values().index(args)]
+            fiscal_classification_id = False
+            for k, v in classifications_keys.items():
+                if v == arg_list:
+                    fiscal_classification_id = k
+                    break
+            template.fiscal_classification_id = fiscal_classification_id
```
Comment (PierrickBrun) [inline account_product_fiscal_classification/__init__.py:53]: :+1:

**Comment by legalsylvain** [comment]
thanks for your review. /ocabot merge

**Pull request merged**